README.md

```diff
@@ -49,7 +49,7 @@ The following assumes the reader knows basics of kubernetes.
 
    2.2 `ingress` in `Ingress.yml` should to be your cluster's loadbalancer IPs.
 
-   2.3 `<SERVICE-NAME>_DOMAIN` variables in `ConfigMap.yml` should to be your domain names.
+   2.3 `<SERVICE-NAME>_DOMAIN` variables in `.env` should to be your domain names.
```

Comment:
Hi,

Please read the entire documentation, if we are using `kubeapply` script why would we want the user to change variables inside the files as well?

Reply:
The default values in .env are domains of openwisp.org like `DASHBOARD_DOMAIN=dashboard.openwisp.org`
the user will need to replace it with their dashboard domain.